app/views/widgets/_smart_banner.html.slim

```diff
@@ -0,0 +1,12 @@
+- if allowed_to_show_smart_banner?
+  .smart-banner
+    .info
+      i.icon-close
+      a href="#{browser.platform.android? ? android_store_page : apple_store_page}"
+        = image_tag "icon", class: "app-icon"
+      h3 Baixe o app da Mudamos
+  javascript:
+    $(".smart-banner .icon-close").click(function() {
```

Comment:
Another #protip: prefer using `on` instead of `click`, as the former creates only one handler for all elements, instead of a handler per element.

```
$(".smart-banner").on("click", ".icon-close", function() {
   ...
});
```